Prepare release 0.32.0 (#2006)
Qiskit/qiskit-ibm-runtime: release-notes/0.32.0.rst

0.32.0 (2024-10-30)
===================

New Features
------------

- Added :func:`~.draw_execution_spans`, a function for creating a Plotly figure that 
  visualizes one or more :class:`~.ExecutionSpans` objects. Also added the convenience

>   method :meth:`~.ExecutionSpans.draw` for invoking the drawing function on a 

Reviewer: ```suggestion
  method :meth:`~.ExecutionSpans.draw` to invoke the drawing function on a 
```

  particular instance. (`1923 <https://github.com/Qiskit/qiskit-ibm-runtime/pull/1923>`__)

  .. code-block:: python

      from qiskit_ibm_runtime.visualization import draw_execution_spans

      # use the drawing function on spans from sampler job data
      spans1 = sampler_job1.result().metadata["execution"]["execution_spans"]
      spans2 = sampler_job2.result().metadata["execution"]["execution_spans"]
      draw_execution_spans(spans1, spans2)

      # convenience to plot just spans1
      spans1.draw() 

- Added a new method, ``backend.refresh()`` that refreshes the
  current backend target with the latest updates from the server. (`1955 <https://github.com/Qiskit/qiskit-ibm-runtime/pull/1955>`__)
- Added :class:`.DoubleSliceSpan`, an :class:`ExecutionSpan` for batching with two slices. (`1982 <https://github.com/Qiskit/qiskit-ibm-runtime/pull/1982>`__)

> - :class:`.SamplerV2`, :class:`.EstimatorV2`, and :class:`.noise_learner.NoiseLearner` now each has

Reviewer: ```suggestion
- Each of :class:`.SamplerV2`, :class:`.EstimatorV2`, and :class:`.noise_learner.NoiseLearner` now has
```

>   a ``backend()`` method that returns the backend the class is configured with. (`1995 <https://github.com/Qiskit/qiskit-ibm-runtime/pull/1995>`__)

Reviewer: ```suggestion
  a ``backend()`` method that returns the backend that the class is configured with. (`1995 <https://github.com/Qiskit/qiskit-ibm-runtime/pull/1995>`__)
```
